frontend/src/scenes/project/Settings/TestAccountFiltersConfig.tsx

```diff
@@ -21,6 +22,25 @@ export function TestAccountFiltersConfig(): JSX.Element {
     return (
         <div className="mb-4 flex flex-col gap-2">
             <div className="mb-4">
+                {!!testAccountFilterWarningLabels && (
```

Comment:
```suggestion
                {!!testAccountFilterWarningLabels && testAccountFilterWarningLabels.length > 0 && (
```

Reply:
Otherwise it'll display this empty state
<img width="2238" alt="Screenshot 2023-01-17 at 09 02 43" src="https://user-images.githubusercontent.com/1851359/212845082-409ee313-56af-40fc-a0db-727874f4c65d.png">